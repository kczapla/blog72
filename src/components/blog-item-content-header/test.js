import React from 'react'
import renderer from 'react-test-renderer'

import BlogItemContentHeader from './index'

describe('blog item content header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<BlogItemContentHeader title='tmp' created='2020-01-01' author='john'/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
