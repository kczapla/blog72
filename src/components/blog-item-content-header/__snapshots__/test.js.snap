// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog item content header renders correctly 1`] = `
<header
  className="sc-bdfBwQ jnglJE"
>
  <h2
    className="sc-dlfnbm eVErIe"
  >
    tmp
  </h2>
  <div
    className="sc-eCssSg kCCemc"
  >
    <time
      dateTime={"2020-01-01T00:00:00.000Z"}
    >
      Jan 01, 2020
    </time>
    <span
      className="sc-gsTCUz ra-dMI"
    >
       by 
    </span>
    <span
      className="sc-hKgILt glBrcB"
    >
      <a
        href="/authors/undefined"
      />
    </span>
  </div>
</header>
`;
